compose-switch is installed as a Linux "alternative" to docker-compose (#4)
docker/compose-switch: README.md

Compose Switch 
--------------

> Compose Switch is a replacement to the Compose V1 `docker-compose` (python) executable. It translate the command line info Compose V2 `docker compose` then run the latter.

Reviewer: ```suggestion
Compose Switch is a replacement to the Compose V1 `docker-compose` (python) executable. It translates the command line into Compose V2 `docker compose` then run the latter.
```

## installation

> We provide an install script for automated installtion:

Reviewer: ```suggestion
We provide a script for automated installation:
```

```console
$ curl -fL https://raw.githubusercontent.com/docker/compose-cli/main/scripts/install/install_linux.sh | sh
```

### Manual installation

1. download compose-switch binary for your architecture
```console
$ curl -fL https://github.com/docker/compose-switch/releases/download/v1.0.1/docker-compose-linux-amd64 -o /usr/local/bin/compose-switch
```
1. make compose-switch executable
```console

> $ chmode +x /usr/local/bin/compose-switch

Reviewer 2: ```suggestion
$ chmod +x /usr/local/bin/compose-switch
```

```
1. rename docker-compose if you already have it installed as `/usr/local/bin/docker-compose` 
```console

> $ mv /usr/local/bin/docker-compose /usr/local/bin/docker-compose-v1

Reviewer 3: Is it possible not to rename it?

Author: we need to define alternative `docker-compose` command that will come first in `PATH`, and unfortunately `/usr/local/bin` comes first (we should not use /usr/local/sbin`)

Reviewer 3: Ha, this is not the one from the package, but one that could have been installed by the user

Author: yes, as documented by https://docs.docker.com/compose/install/

```
1. define an "alternatives" group for `docker-compose` command: 
```console

> $ update-alternatives --install /usr/local/bin/docker-compose docker-compose /$(which docker-compose) 1

Reviewer 3: If you renamed `docker-compose` to  `docker-compose-v1` this should fail

Reviewer 3: It won't fail because the package if any would be on /usr/bin/docker-compose

Reviewer 3: what if the package is not installed?

Compose Switch 
--------------

Compose Switch is a replacement to the Compose V1 `docker-compose` (python) executable. It translates the command line into Compose V2 `docker compose` then run the latter. 

## installation

We provide an script for automated installation:

```console
$ curl -fL https://raw.githubusercontent.com/docker/compose-cli/main/scripts/install/install_linux.sh | sh
```

### Manual installation

1. download compose-switch binary for your architecture
```console
$ curl -fL https://github.com/docker/compose-switch/releases/download/v1.0.1/docker-compose-linux-amd64 -o /usr/local/bin/compose-switch
```
2. make compose-switch executable
```console
$ chmod +x /usr/local/bin/compose-switch
```

> 3. rename `docker-compose` binary if you already have it installed as `/usr/local/bin/docker-compose` 

Reviewer 3: If you installed docker-compose manually, you need to rename it.
If you installed install the docker-compose package from your distro, this step is not necessary

```console
$ mv /usr/local/bin/docker-compose /usr/local/bin/docker-compose-v1
```
4. define an "alternatives" group for `docker-compose` command: 
```console

> $ update-alternatives --install /usr/local/bin/docker-compose docker-compose <PATH_TO_DOCKER_COMPOSE_V1> 1

Reviewer 3: If you installed docker-compose manually, use the path of the renamed file
If you installed install the docker-compose package from your distro, use the patch of docker-compose
